app/views/employees/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: employee) do |form| %>
+  <% if employee.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(employee.errors.count, "error") %> prohibited this employee from being saved:</h2>
+
+      <ul>
+        <% employee.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field mb-3">
+    <%= form.label :first_name %><br>
```

Comment:
tbh I don't like it - as you are allowing users to edit their first & last name then why not allow them to edit their password? IMO this should be the default devise form

Reply:
This is only for setting the name once, users are not able to edit it, this is the case in AC